hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/MergeOnReadIncrementalRelation.scala

```diff
@@ -78,7 +78,16 @@ class MergeOnReadIncrementalRelation(val sqlContext: SQLContext,
   private val tableStructSchema = AvroConversionUtils.convertAvroSchemaToStructType(tableAvroSchema)
   private val maxCompactionMemoryInBytes = getMaxCompactionMemoryInBytes(jobConf)
   private val fileIndex = buildFileIndex()
-
+  private val preCombineField = {
+    val fieldFromTableConfig = metaClient.getTableConfig.getPreCombineField
+    if (fieldFromTableConfig != null) {
```

Comment:
If the field does not exist, will this be an empty string or null?

Reply:
It will be null if it does not exist.